js/views/modals/orderDetail/OrderDetail.js

```diff
@@ -526,8 +529,12 @@ export default class extends BaseModal {
     let showDisputeOrderButton = false;
 
     if (this.buyerId === app.profile.id) {
+      // TODO todo ToDo !!! TODO todo ToDo !!! TODO todo ToDo !!!
+      // todo: when escrow timeout code is ready, include this in the timeoutInfoView
+      // also ensure order is funded if processing error
       showDisputeOrderButton = this.moderatorId &&
```

Comment:
I'll have to circle back to this once #1223 is merged (waiting on server changes) since some of the stuff related to the dispute button was refactored.

I didn't want to merge that PR into this one and have it be some huge diff report.